packages/e2e-tests/test-applications/create-next-app/tests/client/behaviour.test.ts

```diff
@@ -0,0 +1,171 @@
+import { test, expect } from '@playwright/test';
```

Comment:
We should definitely extract this into a helper sometime in the future. Doesn't have to be this PR. I say after we would add this kind of file 2 more times we go for it. The only thing I am worried just now that there is some custom thing we need to do in some framework and we're doing premature DRY.

Reply:
Yes, that makes sense to me 👍